dags/bqetl_fog_decision_support.py

```diff
@@ -55,6 +55,19 @@
         depends_on_past=False,
     )
 
+    wait_for_copy_deduplicate_all = ExternalTaskSensor(
+        task_id="wait_for_copy_deduplicate_all",
+        external_dag_id="copy_deduplicate",
+        external_task_id="copy_deduplicate_all",
+        execution_delta=datetime.timedelta(seconds=10800),
+        check_existence=True,
+        mode="reschedule",
+        allowed_states=ALLOWED_STATES,
+        failed_states=FAILED_STATES,
+        pool="DATA_ENG_EXTERNALTASKSENSOR",
+    )
+
+    fog_decision_support_v1.set_upstream(wait_for_copy_deduplicate_all)
```

Comment:
I wasn't expecting changing the `mozdata` dependencies to have any effect on the Airflow task dependencies, since any views deployed to both `moz-fx-data-shared-prod` and `mozdata` should be exactly the same.

Reply:
So automation missed these dependencies before?

Reply:
Indeed.  I tracked down the bug responsible in the DAG generation code and filed a separate fix for that: #3504